docker-compose-linux.yml

```diff
@@ -96,17 +96,31 @@ services:
     restart: always
     image: minio/minio:latest
     volumes:
-         - ${PWD}/persistent/miniodata:/data:cached
-         - ${PWD}/.bashrc:/root/.bashrc
+      - ${PWD}/persistent/miniodata:/data:cached
+      - ${PWD}/.bashrc:/root/.bashrc
     ports:
          - "9000:9000"
     networks:
       - host-net
       - esmero-net
     environment:
-         MINIO_ACCESS_KEY: minio
-         MINIO_SECRET_KEY: minio123
+      MINIO_ACCESS_KEY: minio
+      MINIO_SECRET_KEY: minio123
     command: server /data
+  mc:
```

Comment:
Can you explain this change, @g7morris ?

Reply:
This allows one to setup the minio container and communication via a new container `mino-client` using a Makefile. Otherwise this would still be a manual step and missing the command line capability that the new container brings.